adapters/adnuntius/adnuntius.go

```diff
@@ -265,6 +108,13 @@ func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters
 		return nil, []error{fmt.Errorf("failed to parse site Ext: %v", err)}
 	}
 
+	var extUser openrtb_ext.ExtUser
+	if ortbRequest.User != nil && ortbRequest.User.Ext != nil {
```

Comment:
Please add a supplemental JSON test case where user.ext is invalid. (if not already added)